thrift-logger/src/main/thrift/thrift_message.thrift

```diff
@@ -2,12 +2,15 @@
 
 namespace java com.pinterest.singer.thrift
 
+include "../../../../audit-library/src/main/thrift/loggingaudit.thrift"
```

Comment:
refactor  modules in singer repo and move thrift related code to `singer-commons`?

Reply:
The audit-library will be used by other components in the logging pipeline, for example Merced.  Thus, I feel it is better to all the code and thrift schema (LoggingAuditHeaders, AuditMessage, AuditConfig) in the audit-library module.

Reply:
if we put it as part of `singer-commons`,  will it prevent Merced etc. from using it?  Merced can declare dependency on singer-commons.  Meanwhile `../../../../audit-library/src/main/thrift/loggingaudit.thrift` shows that the thrift files are not well organized.

Reply:
I agree the thrift files importing does not looks good as a different module is added.  This will not prevent Merced and Sanitizer to use it. Eventually, the merced and Sanitizer will need 2 dependencies, one is singer-commons (including the all configurations classes for Singer), the other one is audit-library.   Initially I thought it is cleaner to have all audit-library related stuff in one module.  If you feel it is better to have singer-commons, we can go this approach.

Reply:
shall we also consider audit-library as part of singer-commons?

Reply:
I feel this is cleaner approach. I will make audit-library part of singer-commons and update the PR.